src/main/java/se/kth/deptrim/core/Trimmer.java

```diff
@@ -77,6 +77,10 @@ public Set<DependencyOriginalAndTrimmed> trimLibClasses(ProjectDependencyAnalysi
           if (finalTrimDependencies.contains(dependencyCoordinates) && !ignoreScopes.contains(key.getScope()) && !dependencyCoordinates.equals(thisProjectCoordinates)) {
             log.info("Trimming dependency " + dependencyCoordinates);
             Set<ClassName> unusedTypes = new HashSet<>(value.getAllTypes());
+            if (value.getUsedTypes().size() == 0) {
```

Comment:
I would substitute the change by `!value.getUsedTypes().isEmpty()` in the `if` statement, as such:

```java
if (!value.getUsedTypes().isEmpty()
              && !finalTrimDependencies.contains(dependencyCoordinates)
              && !ignoreScopes.contains(key.getScope())
              && !dependencyCoordinates.equals(thisProjectCoordinates)) {
...
```

Reply:
Done